src/Lesson_21/Compare.java

```diff
@@ -0,0 +1,13 @@
+package Lesson_21;
+
+import java.util.Comparator;
+
+public class Compare implements Comparator {
+
+    @Override
+    public int compare(Object o1, Object o2) {
+        String object1 = ((Person) o1).getName() + ((Person) o1).getSoname();
```

Comment:
If you parametrize your comparator, it will help you to avoid such unnecessary binding